README.md

```diff
@@ -0,0 +1,70 @@
+Ranch
+=====
+
+Standardized tooling, monitoring, and retry logic for use with Celery
+
+## Installation
+
+Ranch is a Django application. To use Ranch with Celery, add the following to your settings file:
+
+```python
+INSTALLED_APPS = [
+    ...
+    'mbq.ranch'
+]
+
+RANCH = {
+    'env': ENV_NAME,  # e.g. production, development
+    'service': MY_SERVICE_NAME,  # e.g. os-core
```

Comment:
nit: let's encourage a pattern here and give a good example. Also, this is public so maybe don't use os-core?

```python
SERVICE_NAME = 'my-service'
...
    'service': SERVICE_NAME,
...
```